zendframework/zendframework/ZF2014-06.yaml

```diff
@@ -2,12 +2,6 @@ title:     SQL injection vector when manually quoting values for sqlsrv extensio
 link:      http://framework.zend.com/security/advisory/ZF2014-06
 cve:       ~
 branches:
-    2.0.x:
-        time:     2014-09-16 22:00:00
-        versions: [>=2.0.0,<2.0.99]
-    2.1.x:
-        time:     2014-09-16 22:00:00
-        versions: [>=2.1.0,<2.1.99]
```

Comment:
Le 13/03/2015 05:16, Christophe Coevoet a écrit :

> Are 2.0 and 2.1 really not affected by any of these security issues ?

I may indeed have been too quick in this last clean up: at least the
offending code for ZF2014-06 seems actually present in 2.1.8.

Le 13/03/2015 07:27, Christian Flothmann a écrit :

> At least, the blog posts only talk about `2.2` and `2.3`.

Well, AFAICT, there hasn’t been any 2.1 release since April 2013 (nor
any 2.0 releases since Mars 2012), so they were probable EOL, and not
considered upstream anymore when the announce was issued.

I’ll review that last clean up commit ASAP, and will prepare a partial
revert about it, thanks again for the review.

Regards

David